src/lib/resolvers/boolean.ts

```diff
@@ -0,0 +1,13 @@
+import { Identifiers } from '../errors/Identifiers';
+import { err, ok, Result } from '../parsers/Result';
+
+const truths = ['1', 'true', '+', 't', 'yes', 'y'];
+const falses = ['0', 'false', '-', 'f', 'no', 'n'];
```

Comment:
Make it possible for these to be localized. Right now the only option is to simply not use `CoreBoolean` which is not a user-friendly solution at all.

Reply:
> Make it possible for these to be localized. Right now the only option is to simply not use `CoreBoolean` which is not a user-friendly solution at all.

Do you have any example of this in the code? I never worked with i18n so I have no idea how to do that.



> To add to Favna's suggestion, I'd suggest making it possible to supply a list of valid deterministic values on the fly.

Sure why not, although I think adding it to `ArgumentContext` might be "too much" right? as it would be exposed for the context of every arguments, but only boolean needs it. Is this a problem?

Reply:
@Favna have you had the time to look at this? 😄 I just need an example of where i18n is used in the lib, because I never worked with i18n

Reply:
This can be ignored. Marking it as resolved.